packages/flutter_tools/lib/src/ios/bitcode.dart

```diff
@@ -2,65 +2,5 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import '../artifacts.dart';
-import '../base/common.dart';
-import '../base/context.dart';
-import '../base/process.dart';
-import '../base/version.dart';
-import '../build_info.dart';
-import '../globals.dart' as globals;
-import '../macos/xcode.dart';
-
+/// Whether bitcode is enabled by default for iOS builds.
 const bool kBitcodeEnabledDefault = false;
```

Comment:
Can we move this into aot.dart and delete this file?

Reply:
Done